install-extensions.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" -d foodies <<-EOSQL
+CREATE EXTENSION cube;
+CREATE EXTENSION earthdistance;
+EOSQL
```

Comment:
Neat! What are extensions in PostgreSQL?

Reply:
I think they're kind of like modules/libraries that you can import? Tbh didn't know about them until I looked into this task 😛 

From this [blog post](https://severalnines.com/database-blog/creating-new-modules-using-postgresql-create-extension#:~:text=PostgreSQL%20extension%20functionality%20allows%20for,function%20as%20built%2Din%20features.):
> PostgreSQL database has many features and offers a wide range of data types, functions, operators etc. But sometimes it is not enough for some use cases. We can easily extend PostgreSQL's functionality through extensions. This is very useful feature for database developers and administrators.

And in this case it allows us to calculate distances between lat/lon values with simpler syntax (and I think the alternative would be to use some equation like the harversine formula)